### ISSUE TITLE
im gunna rewrite horde stick/wand to work how valve's wand works

### ISSUE THREAD

**Issue by DankBudd** [roberthorvers/Hoard_Mode#85]
apparently the PR i made for stick carrying over charges when upgraded isnt working, which is weird since it worked in all my testing..
anyway, im gunna rewrite it because it will fix problems it has including the carry over charges thing

valves wand gives charges to whichever stick/wand you have had in your inventory the longest
not the wand that has existed the longest, as the code currently does.

i have one question tho @Diellan , do you want me to make gaining charges prioritize wand if you have a stick+wand and the stick is older? 
valves version just goes by held the longest, doesnt prioritize wand.

TL;DR
gunna make horde wand work how valves wand works, should gaining charges prioritize wand if you have both stick and wand?

**Comment by Diellan** [comment]
It should behave as close to Valve's as possible.
